lessons16/index.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+// let admin;
+// let UserName = "Jon";
+// admin = UserName;
+// alert(admin);
+
+
+// let PlanetEarth;
+// let UserName;
+
+
+// let x = 20;
+// let y = 58;
+// let z = 42;
+// let summa = x + y + z;
+// console.log(summa);
+
+
+// let secund = 60;
+// let minute = 60;
+// let hour = 24;
+// let day = 365;
+// let age = 23;
+// age = secund;
+// let myAgeInSeconds = secund * minute * hour * day;
+// alert(myAgeInSeconds);
+
+
+// let count = 42;
+// let userName = '42';
+// count = String(count);
+// userName = Number(userName);
+// console.log(count, userName);
+
+// let a = 1
+// let b = 2
+// let c = "белых медведей";
+// console.log(`${a}${b} ${c}`);
+
+
+// let a = "access";
+// let b = "marine";
+// let c = "frost";
+// let d = "reproach";
+// let e = "chopped";
+
+// var lengthWords = (a.length + b.length + c.length + d.length + e.length);
+// console.log(lengthWords);
+
+
+// let a = '5';
+// let b = "text";
+// let c = true;
+// let at = typeof (a);
+// let bt = typeof (b);
+// let ct = typeof (c);
```

Comment:
Лучше называть переменные полным именем в котором будет понятно что это
В этом случае например так
```suggestion
// const aType = typeof (a);
// const bType = typeof (b);
// const cType= typeof (c);
```